SparkPost/2web2ui
# FE-1242 - Upgrades Matchbox to version 4.6.0

[FE-1242](https://sparkpost.atlassian.net/browse/FE-1242)

### What Changed
- Upgrades Matchbox to version 4.6.0
- For Jest DOM, adds the `window.matchMedia` property to avoid test failures introduced by Matchbox changes
- Adds component wrappers for:
  + `EmptyState` and `EmptyState.LEGACY`
  + `InlineCode`
  + `LabelValue`
- Removes usage of the `Definition` component within Hibana-only features - leaves it in place in features that leverage the OG theme
- Replaces instances of `EmptyState` with `EmptyState.LEGACY` for components that render in both the OG and Hibana themes
- **New:** Adds padding to Dashboard onboarding images - Matchbox removed baked-in padding within its image library

### How To Test
- Ensure the app continues to render as expected
- Ensure test scenarios adequately test changes

### To Do
